components/radiobuttonsform/index.ts

```diff
@@ -0,0 +1,53 @@
+import m from "mithril";
+import RadioButtons, { Attrs as RadioButtonsAttrs } from "../radiobuttons";
+import Label from "../label";
+import Errors, { getErrorClass } from "../errors";
+import { DefaultFormAttrs } from "../types";
+
+
+export const defaultOptions = {
```

Comment:
Same here. For consistency use `DefaultFormAttrs & InputAttrs & { customAttributes }`